test-servers/jenkins/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jenkins:1.642.1
+FROM jenkins:latest
```

Comment:
@amitmawkin | @jdamick  | @tabladrum :  Do we want to allow the Dockerfile to use latest Jenkins image, or does Hygieia require the specified version of the Jenkins image?